common/cairo.c

```diff
@@ -13,6 +13,22 @@ void cairo_set_source_u32(cairo_t *cairo, uint32_t color) {
 			(color >> (0*8) & 0xFF) / 255.0);
 }
 
+cairo_subpixel_order_t to_cairo_subpixel_order(enum wl_output_subpixel subpixel) {
+	switch (subpixel) {
+		case WL_OUTPUT_SUBPIXEL_HORIZONTAL_RGB:
+			return CAIRO_SUBPIXEL_ORDER_RGB;
+		case WL_OUTPUT_SUBPIXEL_HORIZONTAL_BGR:
+			return CAIRO_SUBPIXEL_ORDER_BGR;
+		case WL_OUTPUT_SUBPIXEL_VERTICAL_RGB:
+			return CAIRO_SUBPIXEL_ORDER_VRGB;
+		case WL_OUTPUT_SUBPIXEL_VERTICAL_BGR:
+			return CAIRO_SUBPIXEL_ORDER_VBGR;
+		default:
+			return CAIRO_SUBPIXEL_ORDER_DEFAULT;
```

Comment:
style nit: align case with switch

Reply:
Fixed. I wonder if there's a clang-format rule for that.

Reply:
We've looked into clang-format before, unfortunately there are a few issues with our style that it doesn't support.